driver-core/src/test/java/com/datastax/driver/core/exceptions/ExceptionsTest.java

```diff
@@ -284,7 +284,7 @@ public void should_create_proper_read_timeout_exception() {
     ReadTimeoutException e = new ReadTimeoutException(endPoint1, LOCAL_QUORUM, 2, 3, true);
     assertThat(e.getMessage())
         .isEqualTo(
-            "Cassandra timeout during read query at consistency LOCAL_QUORUM (3 responses were required but only 2 replica responded)");
+            "Cassandra timeout during read query at consistency LOCAL_QUORUM (3 responses were required but only 2 replica responded). In case this was generated during read repair, the consistency level is not representative of the actual consistency.");
```

Comment:
This and the similar change below address some minor fallout from https://github.com/datastax/java-driver/pull/1571

Reply:
Hmmmm... that might actually be desirable anyway.  It makes the test a purer expression of the thing we're actually testing for rather than worrying about the total structure of the log message.  I think I like that.